tests/Browser/pages/CampaignSinglePage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Browser\Pages;
+
+use Laravel\Dusk\Browser;
+
+class CampaignSinglePage extends Page
+{
+    protected $campaignId;
+
+    public function __construct($campaignId)
+    {
+        $this->campaignId = $campaignId;
+    }
+
+    /**
+     * Get the URL for the page.
+     *
+     * @return string
+     */
+    public function url()
+    {
+        return '/campaigns' . $campaignId;
```

Comment:
I think this needs to be `$this->campaignId`!

Reply:
ah! I changed that to `$this->campaignId` and it's still working the same as when it was just `$campaignId` 🤔

Reply:
also realized I was missing another `/` after `'/campaigns` so I'm not sure how it was getting to the page! But I updated to `return '/campaigns/' . $this->campaignId` and still getting the same error that there are no results :-/ It's weird too because the whole time when I was doing the screenshot, it was in fact on the campaign single page (again not sure how it got there with those two errors)